runtime/moonbeam/src/lib.rs

```diff
@@ -1026,10 +1040,11 @@ construct_runtime! {
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 101,
 		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 102,
 		PolkadotXcm: pallet_xcm::{Pallet, Storage, Call, Event<T>, Origin, Config} = 103,
-		Assets: pallet_assets::{Pallet, Call, Storage, Event<T>} = 104,
+		Assets: pallet_assets::<Instance1>::{Pallet, Call, Storage, Event<T>} = 104,
```

Comment:
While I know that now storage prefixes uses the named used in `construct_runtime!` instead of the instance id, are we sure there are no other side effects of chaning which Instance is used? (`()` to `Instance1`)

Reply:
Not that I know. But that does not mean there is unknown situations we could face of course.

Reply:
Changed Instance 1 to () and Instance2 to Instance 1